core/src/toga/command.py

```diff
@@ -320,35 +337,64 @@ def __init__(
         The collection can be iterated over to provide the display order of the commands
         managed by the group.
 
-        :param on_change: A method that should be invoked when this command set changes.
-        :param app: The app this command set is associated with, if it is not the app's
-            own :any:`CommandSet`.
+        The ``in`` operator can be used to evaluate whether a :class:`~toga.Command` is
+        a member of the CommandSet, using either an instance of a Command, or the ID of
+        a command.
+
+        Commands can be retrieved from the CommandSet using ``[]`` notation with the
+        requested command's ID.
+
+        When iterated over, a CommandSet returns :class:`~toga.Command` instances in the
+        their sort order, with :class:`~toga.command.Separator` instances inserted between
```

Comment:
```suggestion
        When iterated over, a CommandSet returns :class:`~toga.Command` instances in
        their sort order, with :class:`~toga.command.Separator` instances inserted between
```

Reply:
Also, this paragraph replaces the existing paragraph above, which should be removed.